Documentation/flexvolume.md

```diff
@@ -78,6 +107,7 @@ This path is commonly used for FlexVolume because `/var/lib/kubelet` is read wri
 You must provide the above found FlexVolume path when deploying the [rook-operator](https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/operator.yaml) by setting the environment variable `FLEXVOLUME_DIR_PATH`.
 For example:
 ```yaml
+env:
```

Comment:
Please add `[...]` on a line after `env:` as other env vars could be "in-between".